examples/getAnnouncementsForUser/.env.example

```diff
@@ -0,0 +1 @@
+RPC_URL='Your rpc url'
```

Comment:
Is this right?
```suggestion
RPC_URL='Your rpc url'
TEST_STEALTH_META_ADDRESS_URI=
TEST_SPENDING_PUBLIC_KEY=
TEST_SPENDING_PRIVATE_KEY=
TEST_VIEWING_PUBLIC_KEY=
TEST_VIEWING_PRIVATE_KEY=
```

Reply:
If it is, also need to update the main `.env.example`

Reply:
removed the irrelevant test env vars